<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="5" column="7">
     <widget class="QPushButton" name="pushButton_div">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_open_braket">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>(</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_mod">
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>mod</string>
      </property>
     </widget>
    </item>
    <item row="4" column="7">
     <widget class="QPushButton" name="pushButton_plus">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="pushButton_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_ln">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>ln</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_clean">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>clean</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QPushButton" name="pushButton_mul">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="pushButton_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QPushButton" name="pushButton_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="pushButton_9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButton_tan">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>tan</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6" colspan="2">
     <widget class="QPushButton" name="pushButton_equal">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QPushButton" name="pushButton_power">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>^</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_cos">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>cos</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_backspace">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>backspace</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_sqrt">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>sqrt</string>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QPushButton" name="pushButton_minus">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="pushButton_7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton_arcsin">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>asin</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton_sin">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>sin</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QPushButton" name="pushButton_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_log">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>lg</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButton_arccos">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>acos</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="pushButton_arctan">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>atan</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="pushButton_dot">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QPushButton" name="pushButton_0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QPushButton" name="pushButton_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="pushButton_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="pushButton_8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_close_braket">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string>)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QLabel" name="label_main">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="4" colspan="4">
     <widget class="QLabel" name="label_result">
      <property name="styleSheet">
       <string notr="true">     background-color: rgb(138, 203, 192);
     border-style: outset;
     border-width: 5px;
     border-radius: 20px;
     border-color: beige;
     font: bold 40px;
     padding: 6px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
